scripts/launcher/lib/virtual_controller.py

```diff
@@ -303,7 +303,7 @@ def run(self):
             self._prepare_and_run()
         except InstallError as e:
             log.error("ERROR: Image creation failed: %s", e)
-            return False
+            return e
```

Comment:
I think you want to raise the exception above or resturn just a `str(e)` instead.